XML/TestCases/FunctionalTests-TVM.xml

```diff
@@ -10,27 +10,28 @@
     <Priority>1</Priority>
     <SetupConfig>
       <SetupType>OneVM</SetupType>
-      <OverrideVMSize>Standard_D2s_v3,=~.*</OverrideVMSize>
+		<SecureBoot>true</SecureBoot>
+		<vTPM>true</vTPM>
+		<OverrideVMSize>Standard_D2s_v3,=~.*</OverrideVMSize>
       <VMGeneration>2</VMGeneration>
     </SetupConfig>
   </test>
   <test>
-    <testName>TVM-TEST-COMPATIBILITY</testName>
-    <testScript>TVM-TestCompatibility.sh</testScript>
-    <files>.\Testscripts\Linux\TVM-TestCompatibility.sh,.\Testscripts\Linux\utils.sh</files>
+    <testName>TVM-TEST-MEASUREDBOOT-COMPATIBILITY</testName>
+    <testScript>TVM-TestMeasuredBootCompatibility.sh</testScript>
+    <files>.\Testscripts\Linux\TVM-TestMeasuredBootCompatibility.sh,.\Testscripts\Linux\utils.sh</files>
     <Platform>Azure</Platform>
     <Category>Functional</Category>
     <Area>TVM</Area>
-    <Tags>tvm</Tags>
+    <Tags>measuredboot</Tags>
```

Comment:
this tag is used for case selection and management, not used in template, can this case pass without updating deployment  template which Vinay asked?

Reply:
Yeah this TestCase would pass without the update Vinay requested.